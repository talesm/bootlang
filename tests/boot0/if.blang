let a = 1;
if a.equals(1) {
    writeln('Ok1');
}
if a.equals(2) {
    writeln('Not Ok');
}
if a.equals(1) {
    writeln('Ok2');
} else {
    writeln('Not Ok');
}
if a.equals(2) {
    writeln('Not Ok');
} else {
    writeln('Ok3');
}
if a.equals(2) {
    writeln('Not Ok');
} else if a.equals(3) {
    writeln('Not Ok');
} else if a.equals(1) {
    writeln('Ok4');
} else if a.equals(10) {
    writeln('Not Ok');
} else {
    writeln('Not Ok');
}
if a.equals(1) {
    writeln('Ok5');
} else if a.equals(3) {
    writeln('Not Ok');
} else if a.equals(1) {
    writeln('Not Ok');
} else if a.equals(10) {
    writeln('Not Ok');
} else {
    writeln('Not Ok');
}
if a.equals(2) {
    writeln('Not Ok');
} else if a.equals(3) {
    writeln('Not Ok');
} else if a.equals(7) {
    writeln('Not Ok');
} else if a.equals(10) {
    writeln('Not Ok');
} else {
    writeln('Ok6');
}
