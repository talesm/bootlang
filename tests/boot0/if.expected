Ok1
Ok2
Ok3
Ok4
Ok5
Ok6
